cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -66,11 +66,17 @@ func extractExecutionState(
 			OutputDir: outputDir,
 		}
 
+		storageFormatV6Migration := mgr.StorageFormatV6Migration{
+			Log:       log,
+			OutputDir: outputDir,
+		}
+
 		migrations = []ledger.Migration{
-			storageUsedUpdateMigration.Migrate,
 			mgr.PruneMigration,
+			storageFormatV6Migration.Migrate,
+			storageUsedUpdateMigration.Migrate,
+			mgr.PruneMigration, // we need post migration pruning before deploying trie update pruning
```

Comment:
I added this post-migration pruning step, similar to what we had in `v0.22-testnet`:
https://github.com/onflow/flow-go/blob/eaf0e5c1e20cde49b94a7ed6bc1768af19bfd3a8/cmd/util/cmd/execution-state-extract/execution_state_extract.go#L72-L76